docs/ja/feature_mouse_keys.md

```diff
@@ -65,13 +65,21 @@ MOUSEKEY_ENABLE = yes
 |----------------------------|-------|---------------------------------------------------------|
 | `MOUSEKEY_DELAY` | 300 | 移動キーを押してからカーソルが移動するまでの遅延 |
 | `MOUSEKEY_INTERVAL` | 50 | カーソル移動間の時間 |
+| `MOUSEKEY_MOVE_DELTA` | 8 | ステップの大きさ |
```

Comment:
You can consider making this minor `MOUSEKEY_MOVE_DELTA` addition in a separate PR that can go directly into the `master` branch.